homeassistant/components/tami4/config_flow.py

```diff
@@ -0,0 +1,91 @@
+"""Config flow for edge integration."""
```

Comment:
The only test requirement is a 100% test coverage of the config_flow.

You should add a test to this one.